### ISSUE TITLE
Research "wizard" packages for Admin Fines calculator

### ISSUE THREAD

**Issue by jason-upchurch** [fecgov/fec-cms#3414]
**User story:** As a user, I need to be able to calculate accurate fine estimates, so that I can better prepare our committee if a fine is assessed.

### Summary
As part of the improvement of our administrative fine calculator, we would like to consider using a wizard package. This would mean the calculator would require slightly less development work and may allow for a more user-friendly interaction. 

Before we can proceed we need to research and identify what our options are and the possible capabilities. 

### Completion criteria
- [ ] identify options and document capabilities, @JonellaCulmer will be able to pair/comment on the options from a UX/design perspective in anticipation of future work (to include initial mockups)
- [ ] make follow up issue "PI board: Get prototype “basic” logic-free wizard working"

**Comment by jason-upchurch** [comment]
moving to sprint 11.2

**Comment by rfultz** [comment]
Adding what I've discovered:


| Name | URL | Technology | Last Updated | Contributors | Documentation | Notes | Screenshots |
--- | --- | --- | --- | --- | --- | --- | ---
Vuetify | https://vuetifyjs.com/en/components/steppers | Vue | Often? | 9+ professionals | Excellent | Part of a larger framework, like Bootstrap for Vue. Solid support—seems like it's part of their core business | ![image](https://user-images.githubusercontent.com/26720877/71907958-2150db00-313b-11ea-90f8-f7b0061d0cff.png) ![image](https://user-images.githubusercontent.com/26720877/71908077-55c49700-313b-11ea-92fd-d943b7c2bc7d.png)
Vue form wizard | https://madewithvuejs.com/vue-form-wizard | Vue | 2018 | 12 | Excellent | Solid option, well-documented | ![image](https://user-images.githubusercontent.com/26720877/71908197-8d334380-313b-11ea-911c-fda54d2e2827.png)
➥ | https://binarcode.github.io/vue-form-wizard/ | Vue |   |   |   |  | ![image](https://user-images.githubusercontent.com/26720877/71908351-d2577580-313b-11ea-8958-92596a7be872.png)
➥ | https://github.com/BinarCode/vue-form-wizard | Vue |   |   |   |  
Dynamic Form Wizard | https://www.vuescript.com/dynamic-form-wizard-vue-js/ | Vue |   |   |   | Based on Vue Form Wizard | ![image](https://user-images.githubusercontent.com/26720877/71908449-0763c800-313c-11ea-8b68-977bc93eb623.png)
React Step Wizard | https://reactjsexample.com/a-modern-flexible-step-wizard-component-built-for-react/ | React | 2018 | 3 |   | Standard, not sure we could include a nav out of the 'box' | ![image](https://user-images.githubusercontent.com/26720877/71908515-2e21fe80-313c-11ea-843b-c0eec07d8d0a.png)
➥ | https://github.com/jcmcneal/react-step-wizard | React | 2018 | 3 |   |  
React Multi-step | https://reactjsexample.com/responsive-reactjs-multistep-form-component/ | React | last week | 9 |   | Standard. Better option than Step Wizard | ![image](https://user-images.githubusercontent.com/26720877/71908571-485bdc80-313c-11ea-92f1-83ce4845f240.png)
React Stepzilla | https://www.npmjs.com/package/react-stepzilla | React | 2017 | 8 |   | Standard, not updated recently | ![image](https://user-images.githubusercontent.com/26720877/71908626-60336080-313c-11ea-975f-84c137aa0ef6.png)
➥ | https://github.com/newbreedofgeek/react-stepzilla | React | 2017 | 8 |   |  
custom: | https://www.raymondcamden.com/2018/01/29/a-multi-step-form-in-vuejs/ | Vue | N/A | 1 |   | Created from scratch—mostly for reference | ![image](https://user-images.githubusercontent.com/26720877/71908748-9244c280-313c-11ea-84e1-902a88280021.png)

**Comment by rfultz** [comment]
We'd talked about choosing a package or approach that we could use with other forms, wizards, steppers, etc. These are some of the widgets currently in use across our sites: https://docs.google.com/document/d/1ItqZTwBUn9ARhr9ulsfuOpjo1eTKHGXhARtJkVkr3bM/edit?folder=0B1PRnuw_hqg1VldEWGJGYmZRRm8#

**Comment by rfultz** [comment]
Just spoke with Pat. Going to close this ticket now, having looked into various packages and experimented with a couple different approaches.
- Vue is strong and offers quite a lot of functionality with a lost barrier of entry (simple, lightweight)
- Vuetify is the strongest stepper package but relies pretty heavily on material design styles—that we don't use _at all_
- Pure custom development wouldn't necessarily be as flexible as we'd like for other modules

Because we'd have to re-name every style in the Vuetify components to get our own styles to work (and theirs not to appear), we already have a lot of functionality available on our current site, and we'd like to avoid fragmenting our current styles and functionality with a second set of both, we're going to go with a Vue + custom/current approach, skipping Vuetify but still leaning on the easy gains given us by a bare-bones Vue implementation plus our current styles and functionality.
